packages/rum-core/src/common/apm-server.js

```diff
@@ -58,18 +58,23 @@ class ApmServer {
   }
 
   createServiceObject() {
-    var cfg = this._configService
-    var stringLimit = cfg.get('serverStringLimit')
+    const cfg = this._configService
+    const stringLimit = cfg.get('serverStringLimit')
 
-    var serviceObject = {
+    const serviceObject = {
       name: sanitizeString(cfg.get('serviceName'), stringLimit, true),
-      version: sanitizeString(cfg.get('serviceVersion'), stringLimit, false),
+      version: sanitizeString(cfg.get('serviceVersion'), stringLimit),
       agent: {
-        name: cfg.get('agentName'),
-        version: cfg.get('agentVersion')
+        name: sanitizeString(cfg.get('agentName'), stringLimit),
+        version: sanitizeString(cfg.get('agentVersion'), stringLimit)
       },
       language: {
         name: 'javascript'
+      },
+      environment: sanitizeString(cfg.get('environment'), stringLimit),
+      runtime: {
+        name: 'browser',
+        version: ''
```

Comment:
It seems strange to send an empty string for version. I know it's required but maybe runtime is not a good fit for the RUM agent either. 

I think we should postpone adding runtime until kibana starts using it. WDYT?

Reply:
Kibana 6.6 already uses this value. This is addressed in next PR with 'N/A'.  I would just send it since  we are touching the metadata anyway.

Reply:
Sorry, I meant Kibana doesn't uses it for the RUM agent.

Reply:
Cool, but that shouldn't stop us from reporting N/A right? Since we are already working on the metadata stuff, I would just keep it as N/A since we would need to do it anyway in future.

Reply:
I'm not sure if we would need to do it in the future, since it's not used in the UI for the RUM agent we might never actually need it and if that's the case we just have added something to the payload that will never be used.

Reply:
Kibana 6.6. UI, 
<img width="1038" alt="screenshot 2019-03-05 at 16 25 39" src="https://user-images.githubusercontent.com/3902525/53816268-9378f200-3f63-11e9-80d4-89776136a8ab.png">

Reply:
Removed runtime from the payload.